docs/operations/hotplug_interfaces.md

```diff
@@ -81,6 +81,7 @@ to the following snippet:
 ```bash
 virtctl addinterface vm-fedora --network-attachment-definition-name new-fancy-net --name dyniface1
 ```
+This will add the interface to the running VMI object, and also mutate the VM spec template.
```

Comment:
Maybe it is worth focusing on the VM and only then adding that "as a consequence", the running VMI is updated.

Reply:
Changed to- "This will add the interface and network to the VM spec template and to the running VMI object.
".